manifests/reposync-crd.yaml

```diff
@@ -279,6 +279,20 @@ spec:
                     - gcpserviceaccount
                     - none
                     type: string
+                  caCertSecretRef:
+                    description: caCertSecretRef specifies the name of the secret
+                      where the CA certificate is stored. The creation of the secret
+                      should be done out of band by the user and should store the
+                      certificate in a key named "cert". For RepoSync resources, the
```

Comment:
Do we talk about both RootSyncs and RepoSyncs in other fields like this? Or should we use different field descriptions for each?

Reply:
Yes, this is written similarly to the existing git field